go.mod

```diff
@@ -23,7 +25,7 @@ require (
 	github.com/onsi/gomega v1.33.0
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0
-	github.com/operator-framework/api v0.23.0
+	github.com/operator-framework/api v0.23.1-0.20240429145053-7d638a091922
```

Comment:
I'm cutting a release of the api so we can update this reference to v0.24.0

Reply:
done!! =D https://github.com/operator-framework/api/releases/tag/v0.24.0

Reply:
Done :)